src/test/java/at/ac/tuwien/kr/alpha/api/externals/stdlib/AspStandardLibraryTest.java

```diff
@@ -139,4 +156,21 @@ public void programWithStringStuff() throws IOException {
 		}
 	}
 
+	@Test
+	@SuppressWarnings("unchecked")
+	public void negatedExternal() throws IOException {
+		Alpha alpha = new Alpha();
+		Program prog = alpha.readProgram(InputConfig.forString(NEGATED_EXTERNAL_ASP));
+		Set<AnswerSet> answerSets = alpha.solve(prog).collect(Collectors.toSet());
+		// Verify every result string has length 6 and contains "foo"
```

Comment:
Also verify that there is the expected number of answer sets!

Reply:
fixed